config.sample.py

```diff
@@ -1,5 +1,20 @@
 import logging
 
+stats = {
+    # stats settings
+    # settings in this section will take effect on the fly. no need to restart
+    # ------------------------------------------------------------------------
+
+    # how long to sleep in seconds between stats reports
+    'sleep_time': 300,
+
+    # print header every nth report
+    'header_every_nth': 20,
+
+    # write a separate .csv file for use in excel.
+    'write_csv': True,
```

Comment:
stdout.  if you enable write_csv, it also writes to <logging_dir>/stats.csv.

logging_dir = os.path.dirname(config.log.get('logging_file'))
csv_path = os.path.join(logging_dir, "stats.csv")